### ISSUE TITLE
Encoding bug?

### ISSUE THREAD

**Issue by jvalrog** [sharkdp/fd#800]
I have some files with strange characters in their names and while **find** can work well with them, **fdfind** won't.

For example:

```
$ find . -type f -exec ls {} \;
'./A3, A4, A5.pdf'
./flujo.docx
'./A2.-TRANSPORTE Y FLUJO DE MATERIALES.pdf'
./a1.docx
'./INDICE PRIMER TRIMESTRE.pdf'
'./a1 preguntas.doc'
'./ejercicios stock.docx'
'./gestion de stocks.pdf'
'./A1.-Gesti'$'\363''n de la producci'$'\363''n en carpinter'$'\355''a y mueble.pdf'
'./Gesti'$'\363''n de la producci'$'\363''n en carpinter'$'\355''a y mueble.pdf'
```
```
$ fdfind -t f --exec ls {}
A3, A4, A5.pdf
A2.-TRANSPORTE Y FLUJO DE MATERIALES.pdf
INDICE PRIMER TRIMESTRE.pdf
a1.docx
a1 preguntas.doc
flujo.docx
ejercicios stock.docx
gestion de stocks.pdf
ls: cannot access 'A1.-Gesti�n de la producci�n en carpinter�a y mueble.pdf': No such file or directory
ls: cannot access 'Gesti�n de la producci�n en carpinter�a y mueble.pdf': No such file or directory
```

**Describe what you expected to happen:**
I expected fdfind to work in this case.

**What version of `fd` are you using?**
fd 7.4.0

**Which operating system / distribution are you on?**
Ubuntu 20.04

**Comment by jvalrog** [comment]
I'm closing this since I tried the last version (8.2.1) and it works as expected. Sad that Ubuntu ships with an old version.

**Comment by sharkdp** [comment]
Yes, this has been fixed in v8.0.0: https://github.com/sharkdp/fd/blob/708efaa812dc81409b7e30c5572a939485a19fdb/CHANGELOG.md#bugfixes-4
